src/js/Controllers/TTSController.js

```diff
@@ -174,8 +174,7 @@ class TTSController {
 
                 this.listener.send(RpcFactory.TTSStartedNotification());
                 this.playNext();
-                if(!rpc.params.speakType.includes('ALERT'))
-                    this.timers[rpc.id] = setInterval(this.onResetTimeout, 9000, rpc.params.appID, "TTS.Speak");
+                this.timers[rpc.id] = setInterval(this.onResetTimeout, 9000, rpc.id);
```

Comment:
```suggestion
                if(!rpc.params.speakType.includes('ALERT'))
                    this.timers[rpc.id] = setInterval(this.onResetTimeout, 9000, rpc.id);
```

Why reset TTS Speak when an Alert is pending?

Reply:
@YuriiYefimenko we have closed #444, can you please make adjustments to ensure onResetTimeout is not sent when the TTS Speak is part of an Alert or AlertManeuver or apply the above suggestion? Thank you!